django/contrib/auth/hashers.py

```diff
@@ -172,6 +174,14 @@ class BasePasswordHasher:
     """
     algorithm = None
     library = None
+    salt_entropy = 71
+
+    @cached_property
+    def char_count(self):
+        # A string of length:
+        # N / log_2(26+26+10)
+        # provides N bits of entropy.
+        return math.ceil(self.salt_entropy / math.log(62, 2))
```

Comment:
I discussed this Florian and we agreed that `cached_property` is an unnecessary in this case, please move it back to the `salt`:
```python
def salt(self):
    """Generate a cryptographically secure nonce salt in ASCII."""
    # A string of length: N / log_2(26+26+10), provides N bits of entropy.
    char_count = math.ceil(self.salt_entropy / math.log(62, 2))
    return get_random_string(char_count) 
```
Sorry for a round-trip.

Reply:
All good. What was the reason for dropping the cached_property?